<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><meta name="description" content='Data from Zhang et al. 2022 which combines data from 10 immunotherapy cohorts.
Especially they used the following cohorts for their training and validation sets:
 - Braun 2020  Braun DA, Hou Y, Bakouny Z, Ficial M, Sant’ Angelo M, Forman J, et al. Interplay of somatic alterations and immune infiltration modulates response to PD-1 blockade in advanced clear cell renal cell carcinoma. Nat Med. 2020;26:909–18. https://doi.org/10.1038/s41591-020-0839-y
 - Mariathasan 2018 Mariathasan S, Turley SJ, Nickles D, Castiglioni A, Yuen K, Wang Y, et al. TGFβ attenuates tumour response to PD-L1 blockade by contributing to exclusion of T cells. Nature. 2018;554:544–8. https://doi.org/10.1038/nature25501
 - Liu 2019  Liu D, Schilling B, Liu D, Sucker A, Livingstone E, Jerby-amon L, et al. Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade in patients with metastatic melanoma. Nat Med. 2019;25. https://doi.org/10.1038/s41591-019-0654-5
 - Gide 2019 Gide TN, Quek C, Menzies AM, Tasker AT, Shang P, Holst J, et al. Distinct Immune Cell Populations Define Response to Anti-PD-1 Monotherapy and Anti-PD-1/Anti-CTLA-4 Combined Therapy. Cancer Cell. 2019;35:238-255.e6. https://doi.org/10.1016/j.ccell.2019.01.003
 - Riaz 2017 Riaz N, Havel JJ, Makarov V, Desrichard A, Urba WJ, Sims JS, et al. Tumor and Microenvironment Evolution during Immunotherapy with Nivolumab. Cell. 2017;171:934-949.e16. https://doi.org/10.1016/j.cell.2017.09.028

The separate, independent testing cohort comes from the following publications:
 - Zhao 2019 Zhao J, Chen AX, Gartrell RD, Silverman AM, Aparicio L, Chu T, et al. Immune and genomic correlates of response to anti-PD-1 immunotherapy in glioblastoma. Nat Med. 2019;25:462–9. https://doi.org/10.1038/s41591-019-0349-y
 - Snyder 2017 Snyder A, Nathanson T, Funt SA, Ahuja A, Buros Novik J, Hellmann MD, et al. Contribution of systemic and somatic factors to clinical response and resistance to PD-L1 blockade in urothelial cancer: An exploratory multi-omic analysis. Minna JD, editor. PLOS Med. 2017;14:e1002309. https://doi.org/10.1371/journal.pmed.1002309
 - Hugo 2016  Hugo W, Zaretsky JM, Sun L, Song C, Moreno BH, Hu-Lieskovan S, et al. Genomic and Transcriptomic Features of Response to Anti-PD-1 Therapy in Metastatic Melanoma. Cell. 2016;165:35–44. https://doi.org/10.1016/j.cell.2016.02.065
 - Van 2015 Van Allen EM, Miao D, Schilling B, Shukla SA, Blank C, Zimmer L, et al. Erratum for the Report “Genomic correlates of response to CTLA-4 blockade in metastatic melanoma” by E. M. Van Allen, D. Miao, B. Schilling, S. A. Shukla, C. Blank, L. Zimmer, A. Sucker, U. Hillen, M. H. Geukes Foppen, S. M. Goldinger, J. Utikal, J. C. Ha. Science. 2016;352:aaf8264–aaf8264. https://doi.org/10.1126/science.aaf8264
 - Kim 2018 Kim ST, Cristescu R, Bass AJ, Kim K-M, Odegaard JI, Kim K, et al. Comprehensive molecular characterization of clinical responses to PD-1 inhibition in metastatic gastric cancer. Nat Med. 2018;24:1449–58. https://doi.org/10.1038/s41591-018-0101-z

Find an extensive description of the data in the original publications and in
Zhang2022, supplementary table S3. https://static-content.springer.com/esm/art%3A10.1186%2Fs13073-022-01050-w/MediaObjects/13073_2022_1050_MOESM1_ESM.pdf
Following the analysis supplied in figshare https://figshare.com/articles/online_resource/Integrated_analysis_of_single-cell_and_bulk_RNA_sequencing_data_reveals_a_pan-cancer_stemness_signature_predicting_immunotherapy_response/17654633
The training data was established by using "ios.Rdata", which is the data from the 10 cohorts combined.
The authors did not give more information how the data was gathered exactly.
The data was preprocessed with sva::ComBat()  (default parameters) to remove batch effects.
Data from the first 5 described cohorts was split randomly into
training (80%, zhang2022_training) and
validation (20%, zhang2022_validation) sets.
The remaining 5 cohorts were used as independent testing set (zhang2022_testing).


Apart from the gene expression data, they established a Stem.Sig gene signature
which consists of selected genes (no coefficients) that were
"correlated with CytoTRACE scores" in 34 scRNA-seq datasets.'><title>Zhang2022 bulk data — zhang2022 • dataMelanoma</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Zhang2022 bulk data — zhang2022"><meta property="og:description" content='Data from Zhang et al. 2022 which combines data from 10 immunotherapy cohorts.
Especially they used the following cohorts for their training and validation sets:
 - Braun 2020  Braun DA, Hou Y, Bakouny Z, Ficial M, Sant’ Angelo M, Forman J, et al. Interplay of somatic alterations and immune infiltration modulates response to PD-1 blockade in advanced clear cell renal cell carcinoma. Nat Med. 2020;26:909–18. https://doi.org/10.1038/s41591-020-0839-y
 - Mariathasan 2018 Mariathasan S, Turley SJ, Nickles D, Castiglioni A, Yuen K, Wang Y, et al. TGFβ attenuates tumour response to PD-L1 blockade by contributing to exclusion of T cells. Nature. 2018;554:544–8. https://doi.org/10.1038/nature25501
 - Liu 2019  Liu D, Schilling B, Liu D, Sucker A, Livingstone E, Jerby-amon L, et al. Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade in patients with metastatic melanoma. Nat Med. 2019;25. https://doi.org/10.1038/s41591-019-0654-5
 - Gide 2019 Gide TN, Quek C, Menzies AM, Tasker AT, Shang P, Holst J, et al. Distinct Immune Cell Populations Define Response to Anti-PD-1 Monotherapy and Anti-PD-1/Anti-CTLA-4 Combined Therapy. Cancer Cell. 2019;35:238-255.e6. https://doi.org/10.1016/j.ccell.2019.01.003
 - Riaz 2017 Riaz N, Havel JJ, Makarov V, Desrichard A, Urba WJ, Sims JS, et al. Tumor and Microenvironment Evolution during Immunotherapy with Nivolumab. Cell. 2017;171:934-949.e16. https://doi.org/10.1016/j.cell.2017.09.028

The separate, independent testing cohort comes from the following publications:
 - Zhao 2019 Zhao J, Chen AX, Gartrell RD, Silverman AM, Aparicio L, Chu T, et al. Immune and genomic correlates of response to anti-PD-1 immunotherapy in glioblastoma. Nat Med. 2019;25:462–9. https://doi.org/10.1038/s41591-019-0349-y
 - Snyder 2017 Snyder A, Nathanson T, Funt SA, Ahuja A, Buros Novik J, Hellmann MD, et al. Contribution of systemic and somatic factors to clinical response and resistance to PD-L1 blockade in urothelial cancer: An exploratory multi-omic analysis. Minna JD, editor. PLOS Med. 2017;14:e1002309. https://doi.org/10.1371/journal.pmed.1002309
 - Hugo 2016  Hugo W, Zaretsky JM, Sun L, Song C, Moreno BH, Hu-Lieskovan S, et al. Genomic and Transcriptomic Features of Response to Anti-PD-1 Therapy in Metastatic Melanoma. Cell. 2016;165:35–44. https://doi.org/10.1016/j.cell.2016.02.065
 - Van 2015 Van Allen EM, Miao D, Schilling B, Shukla SA, Blank C, Zimmer L, et al. Erratum for the Report “Genomic correlates of response to CTLA-4 blockade in metastatic melanoma” by E. M. Van Allen, D. Miao, B. Schilling, S. A. Shukla, C. Blank, L. Zimmer, A. Sucker, U. Hillen, M. H. Geukes Foppen, S. M. Goldinger, J. Utikal, J. C. Ha. Science. 2016;352:aaf8264–aaf8264. https://doi.org/10.1126/science.aaf8264
 - Kim 2018 Kim ST, Cristescu R, Bass AJ, Kim K-M, Odegaard JI, Kim K, et al. Comprehensive molecular characterization of clinical responses to PD-1 inhibition in metastatic gastric cancer. Nat Med. 2018;24:1449–58. https://doi.org/10.1038/s41591-018-0101-z

Find an extensive description of the data in the original publications and in
Zhang2022, supplementary table S3. https://static-content.springer.com/esm/art%3A10.1186%2Fs13073-022-01050-w/MediaObjects/13073_2022_1050_MOESM1_ESM.pdf
Following the analysis supplied in figshare https://figshare.com/articles/online_resource/Integrated_analysis_of_single-cell_and_bulk_RNA_sequencing_data_reveals_a_pan-cancer_stemness_signature_predicting_immunotherapy_response/17654633
The training data was established by using "ios.Rdata", which is the data from the 10 cohorts combined.
The authors did not give more information how the data was gathered exactly.
The data was preprocessed with sva::ComBat()  (default parameters) to remove batch effects.
Data from the first 5 described cohorts was split randomly into
training (80%, zhang2022_training) and
validation (20%, zhang2022_validation) sets.
The remaining 5 cohorts were used as independent testing set (zhang2022_testing).


Apart from the gene expression data, they established a Stem.Sig gene signature
which consists of selected genes (no coefficients) that were
"correlated with CytoTRACE scores" in 34 scRNA-seq datasets.'><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">dataMelanoma</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.0.2.9003</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/ggrlab/dataMelanoma/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-topic">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Zhang2022 bulk data</h1>
      <small class="dont-index">Source: <a href="https://github.com/ggrlab/dataMelanoma/blob/HEAD/R/data.R" class="external-link"><code>R/data.R</code></a></small>
      <div class="d-none name"><code>zhang2022.Rd</code></div>
    </div>

    <div class="ref-description section level2">
    <p>Data from Zhang et al. 2022 which combines data from 10 immunotherapy cohorts.
Especially they used the following cohorts for their training and validation sets:</p>
<p></p><div class="sourceCode"><pre><code><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Braun <span class="dv">2020</span>  Braun DA, Hou Y, Bakouny Z, Ficial M, Sant’ Angelo M, Forman J, et al. Interplay of somatic alterations and immune infiltration modulates response to PD<span class="dv">-1</span> blockade <span class="cf">in</span> advanced clear cell renal cell carcinoma. Nat Med. <span class="dv">2020</span>;<span class="dv">26</span><span class="sc">:</span>909–<span class="fl">18.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1038</span><span class="sc">/</span>s41591<span class="dv">-020-0839</span><span class="sc">-</span>y</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Mariathasan <span class="dv">2018</span> Mariathasan S, Turley SJ, Nickles D, Castiglioni A, Yuen K, Wang Y, et al. TGFβ attenuates tumour response to PD<span class="sc">-</span>L1 blockade by contributing to exclusion of T cells. Nature. <span class="dv">2018</span>;<span class="dv">554</span><span class="sc">:</span>544–<span class="fl">8.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1038</span><span class="sc">/</span>nature25501</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Liu <span class="dv">2019</span>  Liu D, Schilling B, Liu D, Sucker A, Livingstone E, Jerby<span class="sc">-</span>amon L, et al. Integrative molecular and clinical modeling of clinical outcomes to PD1 blockade <span class="cf">in</span> patients with metastatic melanoma. Nat Med. <span class="dv">2019</span>;<span class="fl">25.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1038</span><span class="sc">/</span>s41591<span class="dv">-019-0654-5</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Gide <span class="dv">2019</span> Gide TN, Quek C, Menzies AM, Tasker AT, Shang P, Holst J, et al. Distinct Immune Cell Populations Define Response to Anti<span class="sc">-</span>PD<span class="dv">-1</span> Monotherapy and Anti<span class="sc">-</span>PD<span class="dv">-1</span><span class="sc">/</span>Anti<span class="sc">-</span>CTLA<span class="dv">-4</span> Combined Therapy. Cancer Cell. <span class="dv">2019</span>;<span class="dv">35</span><span class="sc">:</span><span class="dv">238-255</span>.e6. https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1016</span><span class="sc">/</span>j.ccell.<span class="dv">2019</span>.<span class="fl">01.003</span></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Riaz <span class="dv">2017</span> Riaz N, Havel JJ, Makarov V, Desrichard A, Urba WJ, Sims JS, et al. Tumor and Microenvironment Evolution during Immunotherapy with Nivolumab. Cell. <span class="dv">2017</span>;<span class="dv">171</span><span class="sc">:</span><span class="dv">934-949</span>.e16. https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1016</span><span class="sc">/</span>j.cell.<span class="dv">2017</span>.<span class="fl">09.028</span></span></code></pre><p></p></div>
<p>The separate, independent testing cohort comes from the following publications:</p>
<p></p><div class="sourceCode"><pre><code><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Zhao <span class="dv">2019</span> Zhao J, Chen AX, Gartrell RD, Silverman AM, Aparicio L, Chu T, et al. Immune and genomic correlates of response to anti<span class="sc">-</span>PD<span class="dv">-1</span> immunotherapy <span class="cf">in</span> glioblastoma. Nat Med. <span class="dv">2019</span>;<span class="dv">25</span><span class="sc">:</span>462–<span class="fl">9.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1038</span><span class="sc">/</span>s41591<span class="dv">-019-0349</span><span class="sc">-</span>y</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Snyder <span class="dv">2017</span> Snyder A, Nathanson T, Funt SA, Ahuja A, Buros Novik J, Hellmann MD, et al. Contribution of systemic and somatic factors to clinical response and resistance to PD<span class="sc">-</span>L1 blockade <span class="cf">in</span> urothelial cancer<span class="sc">:</span> An exploratory multi<span class="sc">-</span>omic analysis. Minna JD, editor. PLOS Med. <span class="dv">2017</span>;<span class="dv">14</span><span class="sc">:</span>e1002309. https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1371</span><span class="sc">/</span>journal.pmed<span class="fl">.1002309</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Hugo <span class="dv">2016</span>  Hugo W, Zaretsky JM, Sun L, Song C, Moreno BH, Hu<span class="sc">-</span>Lieskovan S, et al. Genomic and Transcriptomic Features of Response to Anti<span class="sc">-</span>PD<span class="dv">-1</span> Therapy <span class="cf">in</span> Metastatic Melanoma. Cell. <span class="dv">2016</span>;<span class="dv">165</span><span class="sc">:</span>35–<span class="fl">44.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1016</span><span class="sc">/</span>j.cell.<span class="dv">2016</span>.<span class="fl">02.065</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Van <span class="dv">2015</span> Van Allen EM, Miao D, Schilling B, Shukla SA, Blank C, Zimmer L, et al. Erratum <span class="cf">for</span> the Report “Genomic correlates of response to CTLA<span class="dv">-4</span> blockade <span class="cf">in</span> metastatic melanoma” by E. M. Van Allen, D. Miao, B. Schilling, S. A. Shukla, C. Blank, L. Zimmer, A. Sucker, U. Hillen, M. H. Geukes Foppen, S. M. Goldinger, J. Utikal, J. C. Ha. Science. <span class="dv">2016</span>;<span class="dv">352</span><span class="sc">:</span>aaf8264–aaf8264. https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1126</span><span class="sc">/</span>science.aaf8264</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a> <span class="sc">-</span> Kim <span class="dv">2018</span> Kim ST, Cristescu R, Bass AJ, Kim K<span class="sc">-</span>M, Odegaard JI, Kim K, et al. Comprehensive molecular characterization of clinical responses to PD<span class="dv">-1</span> inhibition <span class="cf">in</span> metastatic gastric cancer. Nat Med. <span class="dv">2018</span>;<span class="dv">24</span><span class="sc">:</span>1449–<span class="fl">58.</span> https<span class="sc">:</span><span class="er">//</span>doi.org<span class="sc">/</span><span class="fl">10.1038</span><span class="sc">/</span>s41591<span class="dv">-018-0101</span><span class="sc">-</span>z</span></code></pre><p></p></div>
<p>Find an extensive description of the data in the original publications and in
Zhang2022, supplementary table S3. https://static-content.springer.com/esm/art%3A10.1186%2Fs13073-022-01050-w/MediaObjects/13073_2022_1050_MOESM1_ESM.pdf</p>
<p>Following the analysis supplied in figshare https://figshare.com/articles/online_resource/Integrated_analysis_of_single-cell_and_bulk_RNA_sequencing_data_reveals_a_pan-cancer_stemness_signature_predicting_immunotherapy_response/17654633</p><ol><li><p>The training data was established by using "ios.Rdata", which is the data from the 10 cohorts combined.
The authors did not give more information how the data was gathered exactly.</p></li>
<li><p>The data was preprocessed with sva::ComBat()  (default parameters) to remove batch effects.</p></li>
<li><p>Data from the first 5 described cohorts was split randomly into
training (80%, <code>zhang2022_training</code>) and
validation (20%, <code>zhang2022_validation</code>) sets.</p></li>
<li><p>The remaining 5 cohorts were used as independent testing set (<code>zhang2022_testing</code>).</p></li>
</ol><p>Apart from the gene expression data, they established a <code>Stem.Sig</code> gene signature
which consists of selected genes (no coefficients) that were
"correlated with CytoTRACE scores" in 34 scRNA-seq datasets.</p>
    </div>

    <div class="section level2">
    <h2 id="ref-usage">Usage<a class="anchor" aria-label="anchor" href="#ref-usage"></a></h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="fu">zhang2022</span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div class="section level2">
    <h2 id="source">Source<a class="anchor" aria-label="anchor" href="#source"></a></h2>
    <p>Zhang, Z., Wang, ZX., Chen, YX. et al. Integrated analysis of single-cell and bulk RNA sequencing data reveals a pan-cancer stemness signature predicting immunotherapy response. Genome Med 14, 45 (2022). https://doi.org/10.1186/s13073-022-01050-w</p>
    </div>

  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Gunther Glehr.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

